play-scala-websocket-example/app/actors/UserActor.scala

```diff
@@ -47,21 +49,20 @@ class UserActor @Inject()(@Assisted id: String, @Named("stocksActor") stocksActo
   }
 
   // If this actor is killed directly, stop anything that we started running explicitly.
-  override def postStop(): Unit = {
-    log.info(marker, s"Stopping actor $self")
+  def postStop(): Unit = {
```

Comment:
IIUC, in Typed, `postStop` is no longer used and, instead, [a `PostStop` signal must be handled on the `Behavior`](https://doc.akka.io/docs/akka/current/typed/actor-lifecycle.html#stopping-actors).